group-generators/helpers/data-providers/poap/factory-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "POAP",
+  "iconUrl": "",
+  "providerClassName": "POAPProvider",
+  "functions": [
+    {
+      "name": "Attenders of specific events",
+      "functionName": "queryEventsTokenOwners",
+      "description": "Return the list of Ethereum accounts that attended a list of POAP events",
+      "args": [
+        {
+          "name": "event Ids",
+          "type": "(number | string)[]",
+          "example": "63400, 57318",
+          "description": "List of POAP events Ids. https://poap.gallery/"
+        }
+      ]
+    },
+    {
+      "name": "Attenders of a specific event",
```

Comment:
Remove for now as we only use multiple input

Reply:
done